scripts/build.py

```diff
@@ -580,20 +683,22 @@ def build_files(source, target, file_hashes, file_formats=None):
         target: str. Path relative to /oppia directory of directory where
             to copy the files and directories.
         file_hashes: dict(str, str). Dictionary of file hashes.
+        build_tasks: deque(thread). A deque that contains all build tasks queued
+            to be processed.
         file_formats: tuple(str) or None. Tuple of specific file formats to be
             built. If None then all files within the source directory will be
             built.
     """
     print 'Processing %s' % os.path.join(os.getcwd(), source)
     print 'Generating into %s' % os.path.join(os.getcwd(), target)
-    ensure_directory_exists(target)
+    if not does_directory_tree_exist(target):
```

Comment:
You know, the original ensure_directory_exists construct (which checks to see if the dir exists, and creates it if not) might be better. I'm seeing this "if" condition a lot and it clutters up the code a bit...

Reply:
I agree, reverting my changes.